src/alfasim_sdk/_internal/cli.py

```diff
@@ -184,6 +184,37 @@ def package_only(ctx, plugin_dir, package_name, dst):
     )
 
 
+@console_main.command()
+@click.pass_context
+@plugin_dir_option
+def update(ctx, plugin_dir):
```

Comment:
We already have a `cog` task which does something similar. Should we merge them together perhaps?

Reply:
I am sorry. I didn't follow you. 

The `cog` task is not related to this new command that I am adding to the CLI.

Reply:
I meant both tasks get the source code "ready" to use the library. We should merge the two, otherwise the user needs to know which task to execute to update the source.

Could have asked for clarification before hitting merge.

Reply:
`inv cog` generates code for the sdk, is used in development of sdk it self to generate the code that will be used by client code.
`alfasim-sdk update` updates the client code of plugin authors using sdk.

Reply:
Yes my bad.